src/main/java/nextmainfocus/healthcheck/HealthCheckController.java

```diff
@@ -12,10 +12,10 @@ public class HealthCheckController {
 	private static final Logger logger = LoggerFactory.getLogger(HealthCheckController.class);
 
 	@CrossOrigin(origins = "*")
-	@GetMapping("/healthcheck")
+	@GetMapping("/")
```

Comment:
why healthcheck was moved to `/` ? 
best practices?

Reply:
To have something at the root level. It might be replaced some day.